src/js/index.js

```diff
@@ -0,0 +1,143 @@
+import { fetchMenu, storeMenu, removeStoreMenu, updateStoreMenu } from './store/index.js';
+
+const $ = (selector) => document.querySelector(selector);
+
+function App() {
+    let currentCategory;
+
+    const submitMenu = (event) => {
+        event.preventDefault();
+        const keyword = $('#espresso-menu-name').value.trim();
+        if(keyword.length === 0) return;
+        addMenu({name : keyword});
+        storeMenu(keyword, currentCategory);
+
+        updateTotalCount();
+        $('#espresso-menu-name').value = '';
+    }
+
+    const addMenu = (menu) => {
+        $('#espresso-menu-list').insertAdjacentHTML('beforeend', menuTemplate(menu));
+    }
+    
+    const menuTemplate = (item) => {
+        return  `<li class="menu-list-item d-flex items-center py-2">
+        <span class="w-100 pl-2 menu-name ${item.soldOut ? 'sold-out' : ''}">${item.name}</span>
+        <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm mr-1 menu-sold-out-button"
+        >
+        품절
+        </button>
+        <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button"
+        >
+        수정
+        </button>
+        <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm menu-remove-button"
+        >
+        삭제
+        </button>
+    </li>`;
+    }
+    const updateTotalCount = () => {
+        const menuCount = $('#espresso-menu-list').childElementCount;
+        $('.menu-count').innerText = `총 ${menuCount}개`
+    }
+    const updateMenu = (e) => {
+        const $targetList = e.target.closest('li');
+
+        const $menuName = $targetList.querySelector(".menu-name");
+        let updatedMenu = prompt('메뉴명을 수정하세요.', $menuName.innerText) ?? '';
+        updatedMenu = updatedMenu.trim().length > 0 ? updatedMenu : $menuName.innerText;
```

Comment:
```suggestion
  if(updatedMenu.trim().length === 0){
     return;
  }
```

이렇게 변경해도 같은 동작을 하지 않을까요~? 지금은 변경되지 않아도 아래 로직이 동작하면서 안해도 되는 처리까지 진행되고 있는 것 같아용!😮